model/src/main/kotlin/DependencyTreeNavigator.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model
+
+/**
+ * A [DependencyNavigator] implementation based on the classic dependency tree format, which is represented by the
+ * [Project.scopes] property.
+ */
+class DependencyTreeNavigator(
+    /** The underlying [OrtResult]. */
+    val result: OrtResult
```

Comment:
Consider making this private.

Reply:
The class should be private, I agree. It is, however, currently still used by a test helper class from another package. This should no longer be necessary when the expected analyzer test results are reworked.